package.json

```diff
@@ -35,11 +34,14 @@
     "@types/react": "^16.9.51",
     "babel-plugin-transform-react-remove-prop-types": "0.4.24",
     "prop-types": "^15.7.2",
-    "react-dom": "^16.13.1",
     "rollup": "^2.28.2",
     "rollup-plugin-size-snapshot": "^0.12.0",
     "rollup-plugin-terser": "7.0.2"
   },
+  "peerDependencies": {
+    "react": ">=0.14.x",
+    "react-dom": "^16.13.1"
```

Comment:
If we're doing this, then we should set it to the same version as `react`, `>=0.14.x`, that way, the package manager should install the same version.

Reply:
Yup, sounds right, fixed it